cv32e40s/tests/programs/custom/pmp/pmp.c

```diff
@@ -0,0 +1,16 @@
+// #include <stdio.h>
+// #include <stdlib.h>
```

Comment:
If these includes are not needed, might as well remove the comments.

Also, more importantly, we need license headers on all files.
Have a look at other tests and see how license headers are done.